### PR TITLE
Release/1.6.0 beta.1

### PR DESCRIPTION
**Pull request by QuintinWillison**
We need to build and upload the Publishing Example App to Firebase App Distribution.

Without significantly changing or decoupling source code or processes within this repository, the logical, linear path to that is to bump the version (SDK and example app versions are tied together).

This pull request is build atop https://github.com/ably/ably-asset-tracking-android/pull/955 which means that once this PR has landed to `main` and we have pushed the `v1.6.0-beta.1` tag then we can use the manually triggered workflow to publish the app for testers to use via Firebase.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+- Consider supporting static token authentication [\#730](https://github.com/ably/ably-asset-tracking-android/issues/730)
+
+**Fixed bugs:**
+
+- Retry behaviour improvements for Ably API calls [\#927](https://github.com/ably/ably-asset-tracking-android/issues/927)
+- `Publisher.remove\(\)` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
+- Adding a trackable stalls forever if presence.enter\(\) is interrupted by a disconnection [\#896](https://github.com/ably/ably-asset-tracking-android/issues/896)
+- Publisher crashes when location data has NaN value [\#861](https://github.com/ably/ably-asset-tracking-android/issues/861)
+- Subscriber, upon losing connectivity, continues to show the Publisher as online [\#835](https://github.com/ably/ably-asset-tracking-android/issues/835)
+- Subscriber, upon losing connectivity, continues to show the Publisher as online [\#833](https://github.com/ably/ably-asset-tracking-android/issues/833)
```
Comment (KacperKluka) [inline CHANGELOG.md:21]: Duplicated entry 😉

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+
+**Implemented enhancements:**
+
+- Upgrade to Mapbox Nav SDK 2.10.0 [\#882](https://github.com/ably/ably-asset-tracking-android/issues/882)
+- Consider supporting static token authentication [\#730](https://github.com/ably/ably-asset-tracking-android/issues/730)
+
+**Fixed bugs:**
+
+- Retry behaviour improvements for Ably API calls [\#927](https://github.com/ably/ably-asset-tracking-android/issues/927)
+- `Publisher.remove\(\)` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
```
Comment (KacperKluka) [inline CHANGELOG.md:17]: It's rendered as `Publisher.remove\(\)` but should be `Publisher.remove()`, please remove the backslashes 😉

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+- Enable logging in the example apps by default [\#818](https://github.com/ably/ably-asset-tracking-android/issues/818)
+- Update workflows to stop using the `set-output` command [\#817](https://github.com/ably/ably-asset-tracking-android/issues/817)
+- Update workflows to stop using Node.js 12 actions [\#816](https://github.com/ably/ably-asset-tracking-android/issues/816)
+- Refactor Publisher EventQueue to match Subscriber [\#781](https://github.com/ably/ably-asset-tracking-android/issues/781)
+- Consider if we should allow to create multiple publisher instances [\#464](https://github.com/ably/ably-asset-tracking-android/issues/464)
+- Flakey Test: `createAndStartPublisherAndSubscriberAndWaitUntilDataEnds` \(`PublisherAndSubscriberTests`\) [\#259](https://github.com/ably/ably-asset-tracking-android/issues/259)
+
+**Merged pull requests:**
+
+- Fix `publish.stop\(\)` hang when no trackables added [\#952](https://github.com/ably/ably-asset-tracking-android/pull/952) ([jaley](https://github.com/jaley))
```
Comment (KacperKluka) [inline CHANGELOG.md:53]: It's rendered as `Publisher.stop\(\)` but should be `Publisher.stop()`, please remove the backslashes 😉

**Review by KacperKluka — CHANGES_REQUESTED** [review]
A few small things 😉

File: CHANGELOG.md (hunk 1 above)
Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-asset-tracking-android/pull/956/commits/1f35e806a1681ccb3d626a9467364c9739f7aed6

File: CHANGELOG.md (hunk 2 above)
Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-asset-tracking-android/pull/956/commits/b386f86308e7b820a09799fb565fe766daf9ce64

File: CHANGELOG.md (hunk 3 above)
Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-asset-tracking-android/pull/956/commits/b386f86308e7b820a09799fb565fe766daf9ce64

**Review by KacperKluka — APPROVED** [review]
LGTM 👍

**Pull request merged**